liquibase-core/src/main/java/liquibase/GlobalConfiguration.java

```diff
@@ -215,6 +216,11 @@ public class GlobalConfiguration implements AutoloadedConfigurations {
         ALWAYS_DROP_INSTEAD_OF_REPLACE = builder.define("alwaysDropInsteadOfReplace", Boolean.class)
                 .setDescription("If true, drop and recreate a view instead of replacing it.")
                 .build();
+
+        SHOW_UPDATE_SUMMARY = builder.define("showUpdateSummary", Boolean.class)
```

Comment:
This only applies to update/updateSQL so should really be a command argument, not something global.

And then maybe a more generic "showSummary" name to both be less typing, since it's obviously for update and also a term that cam be used for other commands that might want a summary shown someday like rollback or update-checksums or whatever.

Reply:
Instead of being a boolean, should it be an enum to support other formats down the road? (May make just calling it just "summary" or "summarize" even too?) 

Like `summary=none` vs `summary=standard` vs `summary=verbose` vs `summary=json` or whatever?

Reply:
Yes, this should be a command argument.  We are in the process of making the argument be a enum to better control what ouput gets displayed.